packages/react-i18n/src/i18n.ts

```diff
@@ -332,6 +324,39 @@ export class I18n {
 
     return decimal.length === 0 ? DECIMAL_NOT_SUPPORTED : decimal;
   }
+
+  private numberDecimalSymbol() {
+    return this.formatNumber(1.1, {
```

Comment:
This should be replaces with [Intl.NumberFormat.prototype.formatToParts(number)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat/formatToParts) once available.